### ISSUE TITLE
not able to use version 2.0.1 because I have to work with resource files name start with '$' sign

### ISSUE THREAD

**Issue by infahash** [ollide/intellij-java2smali#21]
after updating java2smali to version 2.0.1 I was not able to do my work. it's giving me `Error: '$' is not a valid file-based resource name character: File-based resource names must contain only lowercase a-z, 0-9, or underscore`

on version 1.6 I was able to use the plugin by adding
```
lintOptions {
        abortOnError false
}
```
to my build.gradle. but on the latest version it does not work and giving me the error for resource names

**Comment by ollide** [comment]
Hi @infahash,
could you provide a sample project that leads to your error?

**Comment by infahash** [comment]
It's available on any of decompiled app (XCIPTV app for example). While using resources from decompiled app new version always fails because of resources name error I mentioned

**Comment by ollide** [comment]
The plugin only works when the module or project of the Java/Kotlin file you try to compile to smali can be build.
I assume that you also cannot regularly build the decompiled project?

**Comment by infahash** [comment]
yes. I know. When works on some big projects I have to keep decompiled smali in one project, and making java code on another project. resources are synced between both. so there are a few or many files start with `$` sign. while those files on java project the new version of plugin always failed and works fine when I downgrade.
